lib/api/accounts/signin.js

```diff
@@ -0,0 +1,32 @@
+module.exports = signin
+
+const validUrl = require('valid-url')
+const request = require('request')
+const li = require('li')
+
+function signin () {
+  return (req, res, next) => {
+    if (!validUrl.isUri(req.body.webid)) {
+      return res.status(406).send('This is not a valid URI')
+    }
+
+    request({ method: 'OPTIONS', uri: req.body.webid }, function (err, req) {
+      if (err) {
+        res.status(406).send('Did not find a valid endpoint')
```

Comment:
Why 406 here? I thought that error only dealt with `Accept` headers & content types?